docs/users/overpass-api-guide.md

```diff
@@ -0,0 +1,318 @@
+# Querying OpenStreetMap with Overpass API
+
+Learn how to query real-time OpenStreetMap data using the Overpass API for dynamic, up-to-date visualizations.
+
+## What is Overpass API?
+
+The Overpass API is a read-only API that serves OpenStreetMap (OSM) data. Unlike Overture Maps (which provides pre-processed datasets), Overpass queries the live OSM database, giving you:
+
+- **Real-time data**: Latest OSM edits and contributions
+- **Flexible queries**: Custom filtering with Overpass QL (query language)
+- **Smaller areas**: Best for city-scale queries (not global datasets)
+- **Rich metadata**: Full OSM tags and attributes
+
+## Using the Overpass Operator
+
+The `OverpassOp` operator makes it easy to query OpenStreetMap data. It automatically handles:
+
+- Fetching data from the Overpass API
+- Template replacement for bounding boxes (`{{bbox}}`)
+- Converting OSM JSON to GeoJSON format
+
+### Operator Inputs
+
+- **query**: Overpass QL query string (CodeField with syntax highlighting)
+  - Default query searches for "Central Park" by name (works without bbox)
+  - Use `{{bbox}}` template for bounding box queries
+- **bbox**: Optional bounding box from `BoundsOp` (array of two points: `[[west, south], [east, north]]`)
+  - Only connect if your query uses `{{bbox}}`
+  - Template will be replaced with `(south,west,north,east)` format
+- **endpoint**: Overpass API endpoint (default: `https://overpass-api.de/api/interpreter`)
+- **pulse**: Manual refresh trigger (increment to re-run query)
+
+### Operator Outputs
+
+- **data**: GeoJSON FeatureCollection with all features from the query
+
+## Example: NYC Parks with Overpass
+
+Query all parks in New York City using an interactive bounding box.
+
+### Step 1: Create Bounding Box
+
+Add a `BoundsOp` with NYC coordinates:
+
+- **point1**: `[-74.05, 40.68]` (southwest - Brooklyn)
+- **point2**: `[-73.90, 40.82]` (northeast - Upper Manhattan)
+
+### Step 2: Add Overpass Query
+
+Add an `OverpassOp` node with this query (replace the default):
+
+```
+[out:json][timeout:25];
+(
+  way["leisure"="park"]({{bbox}});
+  relation["leisure"="park"]({{bbox}});
+);
+out geom;
+```
+
+**Important:** Connect the `BoundsOp` output to the `OverpassOp` bbox input. The `{{bbox}}` template will only be replaced when a valid bounding box is connected. Without a bbox connection, queries using `{{bbox}}` will fail.
+
+#### Understanding Overpass QL
```

Comment:
I think this section could be replaced with a link to material they publish to teach the basics instead of us maintaining a syntax guide. It'd kinda be like teaching javascript basics on a codeop guide - it's going to be better to learn JS somewhere else first

Reply:
Maybe instead rewrite this into the same framing as the overture one: Deconstructing the example query